crypto/BRCryptoAccount.c

```diff
@@ -64,28 +98,57 @@ cryptoAccountCreateFromSeedInternal (UInt512 seed,
     return account;
 }
 
-extern UInt512
-cryptoAccountDeriveSeed (const char *phrase) {
-    return cryptoAccountDeriveSeedInternal(phrase);
-}
-
 extern BRCryptoAccount
 cryptoAccountCreate (const char *phrase, uint64_t timestamp) {
     return cryptoAccountCreateFromSeedInternal (cryptoAccountDeriveSeedInternal(phrase), timestamp);
 }
 
 extern BRCryptoAccount
-cryptoAccountCreateFromSeed (UInt512 seed, uint64_t timestamp) {
-    return cryptoAccountCreateFromSeedInternal (seed, timestamp);
-}
+cryptoAccountCreateFromSerialization (const uint8_t *bytes, size_t bytesCount) {
```

Comment:
We don't currently us bytesCount anywhere. Should we parse this defensively and check that value?

Reply:
See the latest commit.  The serialization now starts with  a checksum and a size.  The bytesCount must be at least large enough for those two and then the bytes will be checked against both the checksum and the size.